Resources/Locale/en-US/clothing/components/self-unremovable-clothing-component.ftl

```diff
@@ -0,0 +1 @@
+self-unremovable-clothing = These clothes cannot be removed without outside help.
```

Comment:
Probably should just say "this item" or "this", because there are plenty of things that are technically Clothing, but would be weird to refer to as "clothing" in user-facing text.

Reply:
> Probably should just say "this item" or "this", because there are plenty of things that are technically Clothing, but would be weird to refer to as "clothing" in user-facing text.

Should the component then be renamed to SelfUnremovableItemComponent?

Reply:
Ah, good point!
But I think it's fine to leave the component name as it is, since on a technical level it's still considered Clothing. It's a little bit of a discrepancy between the code and the user-facing text, but I think it's fine.

Reply:
> Ah, good point!
> But I think it's fine to leave the component name as it is, since on a technical level it's still considered Clothing. It's a little bit of a discrepancy between the code and the user-facing text, but I think it's fine.

This component also works if the item is put into the pocket. So I'm not sure that it will always be clothes. Initially, I didn't assume that it would also prohibit removing the item from the pocket, but now it does and I don't think it's a bug. More of a feature or something...

Reply:
It only distinguishes between stripping from/to hands and other slots so far. Only the latter receives the delay. At some point it might be worth expanding this to have a separate delay for each slot, but that might be out of scope for this PR.

Reply:
> It only distinguishes between stripping from/to hands and other slots so far. Only the latter receives the delay. At some point it might be worth expanding this to have a separate delay for each slot, but that might be out of scope for this PR.

The question is not only about the delay, but that all this applies to all equipment slots. Not only for clothing slots. I don't know if this is a problem or not. And if so, I have no idea how to solve it...

Reply:
That's what I meant. A seperate delay for each equipment slot would solve it, since you could just set some of them to 0.